stable_baselines3/common/vec_env/dummy_vec_env.py

```diff
@@ -78,10 +78,10 @@ def close(self) -> None:
         for env in self.envs:
             env.close()
 
-    def get_images(self) -> Sequence[np.ndarray]:
-        return [env.render(mode="rgb_array") for env in self.envs]
+    def get_render_output(self) -> Sequence[Optional[np.ndarray]]:
```

Comment:
I think we should keep `get_images` but raise an error if `render_mode != "rgb_array"`, until gym allows multiple render modes again, see https://github.com/Farama-Foundation/Gymnasium/issues/100

Reply:
Works for me!

Reply:
@araffin , how do you want the render method of the VecEnv to behave then? Should it raise an Error if the render_mode is not rgb_array (in the future if not one of the modes is rgb_array)? Or do you want to the Error in the get_images method?

Reply:
@araffin, gentle ping

Reply:
>  Should it raise an Error if the render_mode is not rgb_array (in the future if not one of the modes is rgb_array)? 

probably a warning would be better? (so a warning and do nothing)
crashing someone's code because one tries to render doesn't look like a user-friendly behavior, no?

Reply:
> I think we should keep get_images but raise an error if render_mode != "rgb_array"

Yes well I was trying to infer what you ment by this. 

I added the warning and reverted the name change. Let me know if there is anything else you would like to see different